src/decanter/core/enums/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Init jobs package"""
+from .evaluators import Evaluator
+from .algorithms import Algos
+
+def check_is_enum(Enum, enum):
```

Comment:
this is not quite how we usually check for enum, but it is ok for now, I'll note this down, we can refactor in some future

Reply:
ok